user_docs/en/changes.md

```diff
@@ -120,6 +123,12 @@ Please refer to [the developer guide](https://www.nvaccess.org/files/nvda/docume
 * It is now possible to redirect objects retrieved from on-screen coordinates, by using the `NVDAObject.objectFromPointRedirect` method. (#16788, @Emil-18)
 * Running SCons with the parameter `--all-cores` will automatically pick the maximum number of available CPU cores. (#16943, #16868, @LeonarddeR)
 * Developer info now includes information on app architecture (such as AMD64) for the navigator object. (#16488, @josephsl)
+* A new utils/urlUtils with different functions to determine link types has been added to check links (#141, @LeonarddeR, @nvdaes)
+* A new INTERNAL_LINK state has been added to controlTypes.states.State (#141, @nvdaes)
+* A new linkType property has been added on `NVDAObject`. It queries the treeInterceptor by default, if any. (#141, @LeonarddeR, @nvdaes)
+* BrowseModeTreeInterceptor object has a new documentUrl property (#141, @LeonarddeR)
+* BrowseModeTreeInterceptor object has a new getLinkTypeInDocument method which accepts an URL to check the link type of the object (#141, @LeonarddeR, @nvdaes)
```

Comment:
can you please group these items and add markdown code formatting around code symbols. The reference should be the PR reference so people can view the code changes directly